pkg/webreg/webreg.go

```diff
@@ -1228,6 +1229,138 @@ const imageExampleLiteral = `- as: repo-e2e
           memory: 2Gi
 `
 
+const quotasAndLeasesPage = `<h2 id="title"><a href="#title">How are Cloud Quota and Aggregate Concurrency Limits Handled?</a></h2>
+<p>
+Jobs that interact with an Infrastructure-as-a-Service (IaaS) cloud provider use credentials shared across the broader
+CI platform. Therefore, all jobs interacting with a specific IaaS will use API quota for these cloud providers from a
+shared pool. In order to ensure that our job throughput for a provider remains within the aggregate limit imposed by
+shared quota, jobs acquire leases for slices of the quota before they run and only relinquish them once all actions are
+completed. This document will describe the mechanism used to provide leases of quota slices to jobs, how jobs determine
+which quota to ask for, how available leases can be configured and how current usage can be investigated.
```

Comment:
`investigated` sounds like there is a lot of effort involved, how about `monitored`?